README.md

```diff
@@ -259,6 +259,16 @@ const User =
     })
 ```
 
+You can check if a field isId accessing the options of the field, like this:
+
+```javascript
+
+const instance = User()
```

Comment:
`const instance = User()` should be `const instance = new User()`?

is this a new instance or a entity type (class)?

Reply:
You're right, I fixed the docs